contact.md

```diff
@@ -22,15 +22,19 @@ The main channels are:
 * #general: General development discussion
 * #random: Non-work-related stuff
 
-### Real-time conference calls
+### Real-Time Conference Calls
```

Comment:
Is title case written down as our standard somewhere?

Reply:
> 
> Is title case written down as our standard somewhere?
Not sure, but all other headings were in title case except for these two that I changed - so I changed these as well to make the page consistent.

Reply:
We follow Google's style guide, which specifies sentence case for titles and headings: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

Reply:
They'll all need to be updated to sentence case, then.

Reply:
Changed accordingly.